governance/roles.md

```diff
@@ -0,0 +1,96 @@
+# Roles within the TAG App Delivery
+
+The TAG includes several key roles that are critical to the group's success.
+Beyond the roles described in this document the TAG has many contributors, all serving in varying capacities.
+
+- [Roles within the TAG App Delivery](#roles-within-the-tag-app-delivery)
+    - [Chair](#chair)
+    - [Technical lead](#technical-lead)
+    - [Working Group Lead](#working-group-lead)
+    - [TOC Liaison](#toc-liaison)
+    - [Emeritus Chair](#emeritus-chair)
+
+## Chair
+
+The (normally three) Co-Chairs of the TAG App Delivery drive the CNCF community efforts of the TAGs domain.
+Chairs are experienced leaders within the CNCF community with rooted expertise in the TAG area. Chairs serve a two-year term but may renew their term by submitting a follow-up application.
```

Comment:
Should we enumerate previous contributions expected of chairs? For example:
- participated in some percentage of TAG meetings
- wrote up a number of project reviews
- contributed to end user papers
- contributed to samples like podtato-head

Reply:
@joshgav: Please write a proposal for this section. In the past, chairs could nominate themselves if they were active in any way. I am not sure if this will help us growing.